scripts/aclshow

```diff
@@ -35,7 +35,7 @@ from natsort import natsorted
 COUNTER_POSITION = '/tmp/.counters_acl.p'
 
 ### acl display header
-ACL_HEADER = ["RULE NAME", "TABLE NAME", "PACKETS COUNT", "BYTES COUNT"]
+ACL_HEADER = ["RULE NAME", "TABLE NAME", "PRIO", "PACKETS COUNT", "BYTES COUNT"]
```

Comment:
Maybe use the full word, `PRIORITY`? It's only 4 more chars.

Reply:
Priority label was "PRIO" long before my changes
https://github.com/Azure/sonic-utilities/pull/442
This time I just reverted this label, as it was before.